src/scripts/site.js

```diff
@@ -287,103 +413,191 @@ var Dashboard = {
+        }
+
         define("multi-download", [embyWebComponentsBowerPath + "/multidownload"], returnFirstDependency);
-        "android" === self.appMode
-            ? define("fileDownloader", ["cordova/filedownloader"], returnFirstDependency)
-            : define("fileDownloader", [embyWebComponentsBowerPath + "/filedownloader"], returnFirstDependency);
+
+        if ("android" === self.appMode) {
+            define("fileDownloader", ["cordova/filedownloader"], returnFirstDependency);
+        } else {
+            define("fileDownloader", [embyWebComponentsBowerPath + "/filedownloader"], returnFirstDependency);
+        }
+
         define("localassetmanager", [apiClientBowerPath + "/localassetmanager"], returnFirstDependency);
-        "cordova" === self.appMode || "android" === self.appMode
-            ? define("castSenderApiLoader", [], getDummyCastSenderApiLoader)
-            : define("castSenderApiLoader", [], getCastSenderApiLoader);
-        self.Windows
-            ? (define("bgtaskregister", ["environments/windows-uwp/bgtaskregister"], returnFirstDependency), define("transfermanager", ["environments/windows-uwp/transfermanager"], returnFirstDependency), define("filerepository", ["environments/windows-uwp/filerepository"], returnFirstDependency))
-            : "cordova" === self.appMode
-                ? (define("filerepository", ["cordova/filerepository"], returnFirstDependency), define("transfermanager", ["filerepository"], returnFirstDependency))
-                : "android" === self.appMode
-                    ? (define("transfermanager", ["cordova/transfermanager"], returnFirstDependency), define("filerepository", ["cordova/filerepository"], returnFirstDependency))
-                    : (define("transfermanager", [apiClientBowerPath + "/sync/transfermanager"], returnFirstDependency), define("filerepository", [apiClientBowerPath + "/sync/filerepository"], returnFirstDependency));
-        "android" === self.appMode
-            ? define("localsync", ["cordova/localsync"], returnFirstDependency)
-            : define("localsync", [apiClientBowerPath + "/sync/localsync"], returnFirstDependency);
-    }
 
-    function getRequirePromise(deps) {
-        return new Promise(function(resolve, reject) {
-            require(deps, resolve)
-        })
+        if ("cordova" === self.appMode || "android" === self.appMode) {
+            define("castSenderApiLoader", [], getDummyCastSenderApiLoader);
+        } else {
+            define("castSenderApiLoader", [], getCastSenderApiLoader);
+        }
+
+        if (self.Windows) {
+            define("bgtaskregister", ["environments/windows-uwp/bgtaskregister"], returnFirstDependency);
+            define("transfermanager", ["environments/windows-uwp/transfermanager"], returnFirstDependency);
+            define("filerepository", ["environments/windows-uwp/filerepository"], returnFirstDependency);
+        } else {
+            if ("cordova" === self.appMode) {
```

Comment:
`deuglify`  doesn't handle nested if's very well, so this nesting should be fixed manually

Reply:
sure, i will modify the cases i caught in this condition. I will arrange cases like this as long as i code. But cases already identified i fix them as you suggest.